Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/UpdateDescription.swift

```diff
@@ -8,6 +8,16 @@ public class UpdateDescription: Codable {
     public let updatedFields: Document
     public let removedFields: [String]
 
+    internal init?(from document: Document) {
+        guard let updatedFields = document["$set"] as? Document,
```

Comment:
if this initializer is actually supposed to exist, this is incorrect. an update description might only have one of `$set` or `$unset`